nservicebus/messaging/messages-events-commands.md

```diff
@@ -78,6 +78,5 @@ We currently have conventions that can identity:
 - [Express messages](/nservicebus/messaging/non-durable-messaging.md)
 - [TimeToBeReceived](/nservicebus/messaging/discard-old-messages.md)
 
-Note: It is important to include the `.Namespace != null`; otherwise a null reference exception will occur during the type scanning.
```

Comment:
Should we also note that using conventions like "Messages" could include types from other unrelated assemblies (eg. the rabbitmq.client has such types) and the fix is the include something like `t.Assembly.Name.StartsWith("MyCorp")`?